engine/src/java/com/runrev/android/Engine.java

```diff
@@ -2231,10 +2231,16 @@ else if (resultCode == Activity.RESULT_OK)
 					// scale to required max width/height
 					float t_width = t_bitmap.getWidth();
 					float t_height = t_bitmap.getHeight();
-					
-					InputStream t_exif_in = ((LiveCodeActivity)getContext()).getContentResolver().openInputStream(t_photo_uri);
-					
-					ExifInterface t_exif = new ExifInterface(t_exif_in);
+                    
+                    ExifInterface t_exif;
```

Comment:
As the rest of this file is indent with tabs, make sure the changes are

There appears to already be an InputStream created for the photo above (`t_in`) so we can resue that here. This means that this code can be a little more compact. I suggest:
```
/* In API Level >= 24, you have to use an input stream to make sure that access
 * to a photo in the library is granted. Before that you can just open the photo's
 * file direct. */
ExifInterface t_exif;
if (Build.VERSION.SDK_INT >= 24)
{
    t_exit = new ExifInterface(t_in);
}
else
{
    t_exif = new ExifInterface(t_photo_uri.getPath());
}
```